src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -560,7 +566,7 @@ def cleanMetadataCatalogFiles(self, transID):
             return res
         fileToRemove = res["Value"]
         if not fileToRemove:
-            self.log.info("No files found for transID %s" % transID)
+            self.log.info("No files found for transID", transID)
```

Comment:
As I'm not sure, is `transID` definitely a string already?

Reply:
Ah we implicitly convert it:

https://github.com/DIRACGrid/DIRAC/blob/9719dadf5272b996ae8ae72e34fb5decd2587558/src/DIRAC/FrameworkSystem/private/standardLogging/Logging.py#L418